src/api/services/NotificationAPI.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ * NotificationAPI Services
+ * Provide services related to Notification (Avvisatura) to communicate with PagoPaAPI
+ */
+
+import * as express from "express";
+import fetch from "node-fetch";
+import { CONFIG } from "../../Configuration";
+import { NotificationSubscriptionRequestType } from "../../enums/NotificationSubscriptionType";
+import { NotificationSubscriptionResponse } from "../types/NotificationSubscriptionResponse";
+
+// Notification Service for PagoPA communications
+export class NotificationAPI {
+  // Update subscription (Activation or Deactivation) to Notification Service for a fiscalCode
+  public static updateSubscription(
+    fiscalCode: string,
+    requestType: NotificationSubscriptionRequestType,
+    res: express.Response,
+    errorCallback: (res: express.Response, errorMsg: string) => void,
+    successCallback: (
+      res: express.Response,
+      notificationSubscriptionResponse: NotificationSubscriptionResponse
+    ) => void
+  ): void {
+    const url =
+      CONFIG.PAGOPA.HOST +
+      ":" +
+      String(CONFIG.PAGOPA.PORT) +
+      CONFIG.PAGOPA.SERVICES.NOTIFICATION_UPDATE_SUBSCRIPTION;
+    const body = {
+      timestamp: Date.now(),
+      requestId: Math.floor(Math.random() * Number.MAX_SAFE_INTEGER),
+      operation:
+        requestType === NotificationSubscriptionRequestType.ACTIVATION
+          ? "A"
```

Comment:
this is very obscure, what are "A" and "D" ?

Reply:
It's a value required by SIA's APIs. I'll add a comment just to explain the meaning